src/Symfony/Component/Workflow/Definition.php

```diff
@@ -17,8 +17,9 @@
 /**
  * @author Fabien Potencier <fabien@symfony.com>
  * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
  */
-class Definition
+final class Definition
```

Comment:
> If it's obvious there is never a need to switch/extend the implementation, keyword final doesn't change the thing

If someone needs to replace/extend implementation, it doesn't mean he is right and extending is correct. You are as a vendor know how to use your tool and you are allowed to introduce some impediments to avoid incorrect usage of your product.